### DISCUSSION TITLE
crops produces more than 1 item when harvested

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#89] [Ideas]
Some crops, like potatoes, spinach, or sunflowers, actually produce many items from one seed in real life. Should we add this sort of functionality to crops in the game? We would probably want to rebalance recipes appropriately, but I think it would give some nice variety to crops and recipes. It would also have some impact on inventory management.

**Comment by jeremyckahn** [comment]
I'll need some time to think through the implications of this change, but I am generally onboard with this idea. Good thinking! 😃
